### PR TITLE
fix(axios-extension): cleanup the baseURL is used to determine if the service was created using a destination

### PR DESCRIPTION
**Pull request by longieirl**
Fix for https://github.com/SAP/open-ux-tools/issues/1257

- the previous flow doesnt handle 400 reponses correctly
- new tests added to ensure destination created ABAP services handle the logger correctly

### CODE REVIEW

File: packages/deploy-tooling/developer-guide.md
Comment (zdravko-georgiev) [inline packages/deploy-tooling/developer-guide.md:1]: We do not need this, right?

Comment (longieirl) [reply packages/deploy-tooling/developer-guide.md:1]: Nope!

**Review by devinea — APPROVED** [review]
code and test changes look good.
Patchset ✅

**Review by cianmSAP — APPROVED** [review]
Changeset ✅
Code changes look good
Have not tested locally

**Pull request merged**